### PR TITLE
📦 scxa-marker-gene-heatmap 💬 Add tests with Cypress

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
Highcharts breaks our tests: anything that goes beyond just checking rendering with static props throws a bunch of exceptions due to the way Highcharts handles updates. They are caused by some mis-handling of refs by js-dom. In fact, [even the tests included in `highcharts-react-official` are failing](https://github.com/highcharts/highcharts-react/issues/384).

After extensive trial and error Cypress is the best thing we’ve found. So here it is! I’ve added [code coverage via Istanbul following the official Cyrpess guide](https://docs.cypress.io/guides/tooling/code-coverage).

### CODE REVIEW

**Review by ke4 — APPROVED** [review]
LGTM 💯

File: packages/scxa-cell-type-wheel/README.md [hunk 1]
```diff
@@ -1,84 +1,14 @@
...
-## Polyfills
-No polyfills are included by default, but you might want one or both of these:
-- [Fetch polyfill](https://github.com/github/fetch)
-- [Babel polyfill](https://babeljs.io/docs/usage/polyfill/)
-
-### NPM
+If you want to omit video recording:
 ```bash
-npm install --save-dev whatwg-fetch @babel/polyfill
+npx cypress run --component
```
Comment (ke4) [inline packages/scxa-cell-type-wheel/README.md:10]: This is the same command as above. Maybe you forgot to add a switch/parameter here?

File: packages/scxa-cell-type-wheel/README.md [hunk 2]
```diff
@@ -1,84 +1,14 @@
...
-  ...
-}
-```
-
-## Run it on your browser
-Use Webpack-Dev-Server:
-```bash
-npx webpack-dev-server -d
-```
+Find coverage reports at:
```
Comment (ke4) [inline packages/scxa-cell-type-wheel/README.md:13]: I think we should add the README on how to achieve coverage reporting. What are the steps to do for this?

**Review by ke4 — CHANGES_REQUESTED** [review]
Tiny change requests

File: packages/scxa-cell-type-wheel/README.md (hunk 2 above)
Comment (alfonsomunozpomer) [reply packages/scxa-cell-type-wheel/README.md:13]: Oh, right!

File: packages/scxa-cell-type-wheel/README.md (hunk 1 above)
Comment (alfonsomunozpomer) [reply packages/scxa-cell-type-wheel/README.md]: Fixed.

**Review by ke4 — APPROVED** [review]
Perfect ;-)  👍

**Pull request merged**